doc/examples/data/plot_specific.py

```diff
@@ -0,0 +1,46 @@
+"""
+===============
+Specific images
+===============
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib
+
+from skimage import data
+
+matplotlib.rcParams['font.size'] = 18
```

Comment:
I've just noticed that you don't set titles in this example. The font config doesn't really make sense then.

Reply:
Other examples look great!

Reply:
It just to keep the tics homogeneous...